.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -4,44 +4,111 @@ about: Use this template to track all release checks for your component
 title: "[Product] Release Checks"
 ---
 
-> [!IMPORTANT]
-> Follow the [guidance on how to use the templates](https://github.com/eclipse-tractusx/sig-release/blob/main/README.md#release-management-acceptance-criteria).
+<!-- 
+Thanks for your contribution! Please fill out this template as good as possible. 
+Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
+Checkout the repository README for process description. 
+-->
 
 ## Release Info
 
 Please provide information on what you want to be included in the Eclipse Tractus-X release.
 If you are not owner of this issue, please provide the information as comment to the issue.
 Make sure to assign this issue to expert(s) for their approval, as soon as you have finished preparation. Multiple assignees allowed; they will un-assign themselves once review completed.
 
-**Version to be included in Eclipse Tractus-X release**: *version placeholder*
+**Responsible contacts**:
+<!-- For this new release -->
+- Committer: *GitHub handle*
+- ExpertGroup: *GitHub handle*
+- Technical contact: *GitHub handle*
```

Comment:
Is this really necessary? Shall we not say Contributors?

Reply:
done